velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -227,6 +228,36 @@ class HiveIcebergTest : public HiveConnectorTestBase {
 
   const static int rowCount = 20000;
 
+ protected:
+  std::shared_ptr<dwrf::Config> config_;
+  std::function<std::unique_ptr<dwrf::DWRFFlushPolicy>()> flushPolicyFactory_;
+
+  std::shared_ptr<ConnectorSplit> makeIcebergSplit(
+      const std::string& dataFilePath,
```

Comment:
nit: `std::string_view dataFilePath`

Reply:
we are sending this argument to HiveIcebergSplit which accepts std::string.